.gitignore

```diff
@@ -14,15 +14,14 @@ classes
 *.ipr
 *.iws
 *.iml
-.idea
+
+.idea/*
+!.idea/codeStyles/
 
 # gradle
 build
 .gradle
 
 # other
 run
-!LICENSE
-!README.*
-!.gitignore
```

Comment:
Why were these removed?

Reply:
It's simply noise. They are added by default.
And make an exception of ignore = add it = default

Reply:
Do you mean that by default `.gitignore` includes these files and these lines aren't necessary to have them be tracked by git? Or do you mean that you think they shouldn't be included in peoples repos?

Reply:
"_these lines aren't necessary to have them be tracked by git_"

Reply:
Thanks, I’ll check that and remove them if they aren’t needed, I seem to remember that they were necessary